site/docs/components/calendar/accessibility.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Keyboard interactions
+
+<KeyboardControls>
+<KeyboardControl keyOrCombos="Tab">
+
+- Tab moves focus to the next focusable element in the calendar.
+
+- If focus is in a day, Tab moves focus to the next focusable component.
```

Comment:
"the next focusable element" here is not the same in the previous line i think? maybe needing a little bit more context